dbcon/mysql/ha_mcs_execplan.cpp

```diff
@@ -9920,7 +9950,7 @@ int getGroupPlan(gp_walk_info& gwi, SELECT_LEX& select_lex, SCSEP& csep, cal_gro
             sc1->columnName(sc->columnName());
             sc1->tableName(sc->tableName());
             sc1->tableAlias(sc->tableAlias());
-            sc1->viewName(lower(sc->viewName()));
+            sc1->viewName(sc->viewName());
```

Comment:
This should have `lower_case_table_names` as the second argument.

Reply:
Anything put into SimpleColumn has been converted if needed. No need to spend cpu cycles doing it again.